tests/sceneManager_test.cpp

```diff
@@ -25,8 +25,9 @@ class SceneManagerTest : public ::testing::Test
         }
         return false;
     }
-
+    
     SceneManager manager;
+    core::Vec3 kVZero;
```

Comment:
This does not belong here. It is a constant and 1. should me marked as such, and since this is quite generic, move it to the top of the file.

Reply:
done

Reply:
I still see it here.

Reply:
hmm weird. I added it here: 
![image](https://user-images.githubusercontent.com/45213495/97059223-23800b00-1590-11eb-961b-76bd88a1321b.png)

Reply:
do you mean even higher to the top of the file?